src/stories/Colours.stories.tsx

```diff
@@ -1,6 +1,50 @@
 import React from 'react';
 import { ColorPalette, ColorItem } from '@storybook/addon-docs/blocks';
 
+import colourTokens from 'build/js/colours';
```

Comment:
I was slightly confused here, because `/build` has been added to `.gitignore`. I'm assuming this is because this refers to `/src/build`, not `/build`. Maybe the two directories could be named differently to avoid confusion ?

Reply:
(that was a thought, it's not too important!)

Reply:
Looks like that was a hangover from some earlier exploration - Style Dictionary defaults to outputting to a build folder in the root. I'll remove it from the ignore file for clarity.